tests/Feature/ProjectTest.php

```diff
@@ -675,10 +676,10 @@
     expect($org_future_project->status)->toEqual('Upcoming');
     expect($indeterminate_project)->started->toBeFalse();
 
-    $this->assertEquals(5, count($organization->projects));
-    $this->assertEquals(2, count($organization->completedProjects));
-    $this->assertEquals(1, count($organization->inProgressProjects));
-    $this->assertEquals(1, count($organization->upcomingProjects));
+    expect(count($organization->projects))->toEqual(5);
+    expect(count($organization->completedProjects))->toEqual(2);
+    expect(count($organization->inProgressProjects))->toEqual(1);
+    expect(count($organization->upcomingProjects))->toEqual(1);
```

Comment:
Here it might be better to use [`toHaveCount()`](https://pestphp.com/docs/expectations#content-tohavecountint-count).

Reply:
This also applies to other instances that use count.